config/jail.conf

```diff
@@ -294,6 +294,12 @@ maxretry = 1
 port    = http,https
 logpath = %(nginx_error_log)s
 
+[nginx-botsearch]
+
+port     = http,https
+logpath  = %(nginx_error_log)s
+maxretry = 10
+findtime = 300
```

Comment:
remove custom findtime, set maxretry to low number like already in apache-botsearch